docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -8,253 +8,230 @@ videoBanner: UFhT52d5bYg
 
 ## Prerequisites
 
-Collect a few pieces of information about your Redshift cluster which you'll
-need to configure access.
+- Teleport version `(=teleport.version=)`.
+- AWS account with Redshift databases and permissions to create and attach IAM
```

Comment:
I think "with Redshift databases" can be more precise here. Is this the same as "a Redshift cluster"?